### ISSUE TITLE
Clean up image src files in tts

### ISSUE THREAD

**Issue by matdillen** [matdillen/legendary-generator#8]
Synchronize the card image files with those used in the shiny app. Easiest to do this is overwriting the filenames and identifiers in the save file.

Once this is done, card data such as card name, vp or cost can readily be tagged onto individual cards by editing the save file.

**Comment by matdillen** [comment]
Image files harmonized for villains, heroes, henchmen and masterminds.

Not done:

Transformed heroes

Out of scope as not present in app:

Horrors
Ambitions
Twists
Strikes
Wounds
Bystanders
